pkg/controller/migmigration/task.go

```diff
@@ -147,7 +147,7 @@ func (t *Task) Run() error {
 			if t.hasPVs() {
 				t.Phase = AnnotateResources
 			} else {
-				t.Phase = Completed
+				t.Phase = QuiesceApplications
```

Comment:
The `QuiesceApplications` phase should only be entered if `t.quiesce()` is true.
I think this section of code should probably read:
```
			if t.hasPVs() {
				t.Phase = AnnotateResources
			} else if t.quiesce() {
				t.Phase = QuiesceApplications
			} else {
				t.Phase = Completed
			}
```

Reply:
Yes!  Good catch.  Corrected.